schematic_db/api_utils/api_utils.py

```diff
@@ -108,38 +110,24 @@ def create_schematic_api_response(
     api_url = getenv("API_URL", "https://schematic.api.sagebionetworks.org/v1/")
     endpoint_url = f"{api_url}/{endpoint_path}"
     start_time = datetime.now(pytz.timezone("US/Pacific"))
+    headers = {"Authorization": f"Bearer {access_token}"}
     try:
-        response = requests.get(endpoint_url, params=params, timeout=timeout)
+        response = requests.get(
+            endpoint_url, params=params, headers=headers, timeout=timeout
```

Comment:
@andrewelamb just wanted to confirm, including the token in the header for endpoints that don't have it defined won't cause issues correct?

Reply:
That's correct. I'm not sure what requests does behind the scenes when the header is 
`{"Authorization": f"Bearer None"}` but all the other endpoints still work.